frontend/src/components/layout/Layout.module.scss

```diff
@@ -7,17 +8,122 @@
 
   // See https://fkhadra.github.io/react-toastify/how-to-style#override-css-variables
   --toastify-toast-width: 95%;
-  --toastify-color-success: #{$color-green-60};
-  --toastify-text-color-success: #{$color-white};
-  --toastify-color-error: #{$color-red-50};
+  --toastify-color-light: #{$color-white};
+  --toastify-color-dark: #{$color-grey-50};
+  --toastify-color-info: #{$color-blue-50};
+  --toastify-color-success: #{$color-green-50};
+  --toastify-color-warning: #{$color-yellow-50};
+  --toastify-color-error: #{$color-red-60};
+
+  --toastify-text-color-info: #{$color-white};
+  --toastify-text-color-success: #{$color-grey-50};
+  --toastify-text-color-warning: #{$color-grey-50};
   --toastify-text-color-error: #{$color-white};
+
+  --toastify-toast-min-height: $layout-md;
+  --toastify-toast-max-height: $layout-md;
 }
 
+// Allows us to dynamically set the background color based on Toast container state
+$toastify-toast-close-button: var(--toast-close-button);
+
 .toast {
-  border-radius: $border-radius-md;
+  @include text-body-sm;
   text-align: center;
-  font-weight: 700;
+  font-weight: 400;
+  line-height: 21px;
+  height: $layout-md;
```

Comment:
Suggestion: I call this out in my general review comment, but if we can let the content (using padding, etc) dictate the toast height, let's. Esp for wordy messages here, on mobile, it'll need to break into two lines.